pkg/csi_driver/controller.go

```diff
@@ -723,7 +751,15 @@ func (s *controllerServer) ControllerExpandVolume(ctx context.Context, req *csi.
 		return nil, file.StatusError(err)
 	}
 	if filer.State != "READY" {
-		return nil, fmt.Errorf("lolume %q is not yet ready, current state %q", volumeID, filer.State)
+		return nil, fmt.Errorf("Volume %q is not yet ready, current state %q", volumeID, filer.State)
+	}
+
+	// if the Filestore instance is in a small band of the tier then verify if the volume expansion
+	// request cross the band size.
+	invalidExpansion := invalidVolumeExpansionRequest(req.GetCapacityRange(), filer.Volume.SizeBytes, filer.Tier)
+	if invalidExpansion {
+		klog.Warningf("Volume expansion not supported beyond small %s band. Please create a new instance with higher storage capacity.", filer.Tier)
```

Comment:
this log is repetitive of the above warningf logging, why not put this message inside the `status.Errorf`